libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -142,67 +142,72 @@ export default class DocElementParser {
     }
 
     // Make sure there are no extra pipes
-    let pipeSplitContent: string[] = tokenItem.text.split('|');
-    pipeSplitContent = pipeSplitContent.map( value => {
+    const pipeSplitContent: string[] = tokenItem.text.split('|').map(value => {
       if (value) {
         return value.trim();
       }
     });
     if (pipeSplitContent.length > 2) {
       documentation.reportError('The {@link} tag contains more than one pipe character ("|")');
-      return;
+      return undefined;
     }
 
+    const addressPart: string = pipeSplitContent.length > 0 ? pipeSplitContent[0] : '';
```

Comment:
>pipeSplitContent.length > 0 [](start = 32, length = 27)

Realistically can pipeSplitContent.length ever be <= 0?  (If so, shouldn't that be an error?)